src/reducers/NewSensorReducer.js

```diff
@@ -33,8 +33,13 @@ export const NewSensorReducer = (state = initialState(), action) => {
       const { configField, configType, value } = payload;
 
       const oldConfig = state[configType];
-      const newConfig = { ...oldConfig, [configField]: value };
 
+      if (configField === 'duration') {
+        const newConfig = { ...oldConfig, [configField]: value * 60 };
+        return { ...state, [configType]: newConfig };
+      }
+
+      const newConfig = { ...oldConfig, [configField]: value };
```

Comment:
this combined with a few changes to duration editor to allow it to be more generic